src/aspire/abinitio/commonline_sync.py

```diff
@@ -24,15 +24,25 @@ class CLSyncVoting(CLOrient3D):
     Journal of Structural Biology, 169, 312-322 (2010).
     """
 
-    def __init__(self, src, n_rad=None, n_theta=None):
+    def __init__(self, src, n_rad=None, n_theta=360, max_shift=0.15, shift_step=1):
```

Comment:
Wondering if we should add `n_check` to `CLSyncVoting`?

Reply:
Actually, there was discussion of removing `n_check` altogether, since it is an unused parameter/setting. Will get confirmation on that. Thanks.